libethashseal/genesis/ropsten.cpp

```diff
@@ -50,10 +50,14 @@ R"E(
 		"gasLimit": "0x1000000"
 	},
 	"accounts": {
-		"0000000000000000000000000000000000000001": { "wei": "1", "precompiled": { "name": "ecrecover", "linear": { "base": 3000, "word": 0 } } },
-		"0000000000000000000000000000000000000002": { "wei": "1", "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } } },
-		"0000000000000000000000000000000000000003": { "wei": "1", "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } } },
-		"0000000000000000000000000000000000000004": { "wei": "1", "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } },
+		"0000000000000000000000000000000000000001": { "precompiled": { "name": "ecrecover", "linear": { "base": 3000, "word": 0 } } },
+		"0000000000000000000000000000000000000002": { "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } } },
+		"0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } } },
+		"0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } },
+		"0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp", "startBlock": "0x19f0a0" } },
+		"0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "startBlock": "0x19f0a0", "linear": { "base": 500, "word": 0 } } },
+		"0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "startBlock": "0x19f0a0", "linear": { "base": 40000, "word": 0 } } },
```

Comment:
would it be better to hardcode this configuration as in the other precompiles? 
`"linear": { "base": 40000, "word": 0 }`

Reply:
do you mean for `alt_bn128_pairing_product` ?

Reply:
I mean hardcode `"linear": { "base": 40000, "word": 0 }`
for `alt_bn128_G1_mul`

so we won't have genesis config issues.

Reply:
Are you sure about the names of the precompiled?